master/internal/db/postgres_experiments.go

```diff
@@ -0,0 +1,47 @@
+package db
+
+import (
+	"encoding/json"
+	"fmt"
+	"sort"
+)
+
+// ExperimentLabels returns a flattened and deduplicated list of all the
+// labels in use across all experiments.
+func (db *PgDB) ExperimentLabels() (labels []string, err error) {
+	// First, assemble all the JSON lists that the database returns into a
+	// single tally of all the labels
+	type dbLabelList struct {
+		Labels []byte
+	}
+	var rawLists []dbLabelList
+	err = db.Query("get_experiment_labels", &rawLists)
```

Comment:
nit: should we check and wrap this?

Reply:
Oh I totally missed that! Yes, we definitely should.